src/components/assets/EvmCbridgeToken.vue

```diff
@@ -124,6 +114,12 @@ export default defineComponent({
       }
     };
 
+    const transferLink = computed<string>(() => {
```

Comment:
This block is repeated a lot. What about making it as a function?

Reply:
I've fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/479/commits/09f01d11f8b9a7892d85517e1d40532136f4430f#diff-003c474de6953460a9c7b1afc9a133e05421b357d4bb201c78d9bb2f7f21f30aR16) commit.